src/org/traccar/model/Position.java

```diff
@@ -291,5 +291,4 @@ public void setNetwork(Network network) {
     public String getType() {
         return super.getType();
     }
-
```

Comment:
Again, revert any unnecessary changes to the file. Please check diff on GitHub yourself before asking to review.